content/docs/setup/kubernetes/install/multicluster/gateways/index.md

```diff
@@ -73,10 +73,17 @@ on **each** Kubernetes cluster.
 1. Run the following commands in **every cluster** to deploy an identical Istio control plane
    configuration in all of them.
 
-    * Create a Kubernetes secret for your generated CA certificates using a command similar to the following. See [Certificate Authority (CA) certificates](/docs/tasks/security/plugin-ca-cert/#plugging-in-the-existing-certificate-and-key) for more details.
+    * Grant cluster administrator (`cluster-admin`) permissions to the current user.
+    On GKE platform, the following command uses `gcloud config` to get the current user.
+    For other platforms, please refer to the [platform setup](/docs/setup/kubernetes/prepare/platform-setup/).
```

Comment:
This doesn't sound right. I didn't need to do anything to run this on IBM Cloud. I'd suggest removing this:
```
    * Grant cluster administrator (`cluster-admin`) permissions to the current user.
    On GKE platform, the following command uses `gcloud config` to get the current user.
    For other platforms, please refer to the [platform setup](/docs/setup/kubernetes/prepare/platform-setup/).

        {{< text bash >}}
        $ kubectl create clusterrolebinding cluster-admin-binding --clusterrole=cluster-admin --user="$(gcloud config get-value core/account)"
        {{< /text >}}
```
and replace it with this:
```
    {{< warning >}}
    Make sure that the current user has cluster administrator (`cluster-admin`) permissions and grant them if not. 
    On the GKE platform, for example, the following command can be used:
    
    {{< text bash >}}
    $ kubectl create clusterrolebinding cluster-admin-binding --clusterrole=cluster-admin --user="$(gcloud config get-value core/account)"
    {{< /text >}

    {{< /warning >}}
```

Reply:
Done.